test/static_string.cpp

```diff
@@ -295,6 +295,43 @@ testTWS(Arithmetic value, const wchar_t* wstr_expected = L"", bool test_expected
   }
 }
 
+// done
+static
+void
+testTypeTraits()
+{
+    {
+        using S = static_string<1>;
```

Comment:
Is 

```
using S = static_string<0>;
```

a thing we need to test?

Reply:
Sure; updated.

Re "That sounds great" — unclear whether you mean "great" in re "open a new PR for trivially relocatable" (which makes sense only if you don't want to take this one — trivially copyable types are trivially relocatable by definition) or just generally expressing contentment with this PR. :)

Reply:
just generally expressing contentment with this PR. :)